renderer/viewer/lib/worldrendererCommon.ts

```
@@ -442,6 +443,11 @@ export abstract class WorldRendererCommon<WorkerSend = any, WorkerReceive = any>
         this.blockStateModelInfo.set(cacheKey, info)
       }
     }
+
+    if (data.type == 'heightmap') {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)